src_c/math.c

```diff
@@ -550,6 +552,28 @@ static PyMemberDef vector_members[] = {
     {NULL} /* Sentinel */
 };
 
+static PyObject *
```

Comment:
Return type here could be `pgVector *` because almost all function calls to this cast this to `pgVector *`